src/js/index.js

```diff
@@ -40,6 +40,22 @@ function printInitialTasks() {
     checkbox.className = 'form-check-input pull-left';
     checkbox.style.marginRight = '17px';
     checkbox.value = task.completed;
+    if (task.completed) {
+      checkbox.checked = true;
+    }
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        console.log('ischecked');
+        task.completed = true;
+        taskManager.updateALL();
```

Comment:
- [ ] Kindly change this and all other functions to es6 arrow function syntax, as stated in the JavaScript best practices guide.

Reply:
:heavy_check_mark: well resolved